src/validation/areChangeFilesDeleted.ts

```diff
@@ -17,12 +17,12 @@ export function areChangeFilesDeleted(options: BeachballOptions): boolean {
     process.exit(1);
   }
 
-  console.log(`Checking for deleted change files against "${branch}"`);
+  console.log(`Checking for deleted and renamed change files against "${branch}"`);
   const changeFilesDeletedSinceRef = getChangesBetweenRefs(
     branch,
     'HEAD',
     [
-      '--diff-filter=D', // showing only deleted files from the diff.
+      '--diff-filter=DR', // showing only deleted and renamed change files from the diff.
```

Comment:
Can we add some unit tests?

Reply:
Sure - I will work on this.

Reply:
+1 for tests. You can find existing tests for this function here: https://github.com/microsoft/beachball/blob/master/src/__e2e__/validation.test.ts#L86

Reply:
Added some unit tests as well.